doc/md/guides/orms/hibernate.mdx

```diff
@@ -0,0 +1,439 @@
+---
+id: hibernate
+title: Automatic migration planning for Hibernate
+slug: /guides/orms/hibernate
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## TL;DR
+* [Hibernate](https://hibernate.org/orm/) is an ORM library that's widely used in the Java community.
+* [Atlas](https://atlasgo.io) is an open-source tool for inspecting, planning, linting and
+  executing schema changes to your database.
+* Developers using Hibernate can use Atlas to automatically plan schema migrations
+  for them, based on the desired state of their schema instead of crafting them by hand.
```

Comment:
> instead of crafting them by hand.

Are you sure this is true for Hibernate?

Reply:
Yes